docs/UserGuide.md

```diff
@@ -210,6 +210,40 @@ Example:
 `markapplicant 106 s/accepted`
 - Marks the status of the applicant with ID 106 as accepted.
 
+`markapplicant 1239 s/interviewed`
+- Marks the status of the applicant with ID 1239 as interviewed.
```

Comment:
Index would be more specific and consistent with the rest of the UG instead of ID here.